core/src/main/scala/org/apache/spark/rpc/netty/Inbox.scala

```diff
@@ -206,6 +206,13 @@ private[netty] class Inbox(
       case NonFatal(e) =>
         try endpoint.onError(e) catch {
           case NonFatal(ee) => logError(s"Ignoring error", ee)
+          case NonFatal(ee) =>
+            if (stopped) {
+            logDebug(s"Ignoring error", ee)
```

Comment:
logInfo, not debug, probably. I can see reducing the level if you know your'e stopping, but not in other cases. The code style here would have to be fixed. For example you don't use string interpolation here.